File: WMS_Medellin/Common/Models/EstructuraUploadModel.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class EstructuraUploadModel
    {
        public string identificador1 { get; set; } = " ";
        public string identificador2 { get; set; } = " ";
        public string cia_Destino { get; set; } = " ";
        public string co_Destino { get; set; } = " ";
        public string tipoDoc_Destino { get; set; } = " ";
        public string numDoc_Destino { get; set; } = " ";
        public int cia_Origen { get; set; } = 0;
        public string co_Origen { get; set; } = " ";
        public string tipoDoc_Origen { get; set; } = " ";
        public int numDoc_Origen { get; set; } = 0;
        public string co_Transf_Salida { get; set; } = " ";

        public int idDocumentoGT { get; set; } = 0;
        public string strDocumentoGT { get; set; } = " ";

        public bool bitError { get; set; } = false;
        public string datosEnviados_GT { get; set; } = " ";
        public string resultado_GT { get; set; } = " ";
        public string detalleResultado_GT { get; set; } = " ";
        public string datosEnviados_UNOEE { get; set; } = " ";
        public string resultado_UNOEE { get; set; } = " ";
        public string detalleResultado_UNOEE { get; set; } = " ";
        public string urlWebServiceUNOEE { get; set; } = " ";
        public string otrosDetalles { get; set; } = " ";
        public string tabla { get; set; } = " ";

        public DataSet dsDatos;
        public DataTable dtPaquetes;
        public DataTable dtDocumentos;
        public DataTable dtMovimientos;
        public DataTable dtDescuentos;
        public DataTable dtCxC;
    }
}

File: WMS_Medellin/Upload_Entrada_POP_Directas_sin_OC/Modelos/EstructuraUploadModel.vb
Public Class EstructuraUploadModel

    Public Property identificador1 As String = " "
    Public Property identificador2 As String = " "
    Public Property cia_Destino As String = " "
    Public Property co_Destino As String = " "
    Public Property tipoDoc_Destino As String = " "
    Public Property numDoc_Destino As String = " "
    Public Property cia_Origen As Integer = 0
    Public Property co_Origen As String = " "
    Public Property tipoDoc_Origen As String = " "
    Public Property numDoc_Origen As Integer = 0
    Public Property bitError As Boolean = False
    Public Property datosEnviados_GT As String = " "
    Public Property resultado_GT As String = " "
    Public Property detalleResultado_GT As String = " "
    Public Property datosEnviados_UNOEE As String = " "
    Public Property resultado_UNOEE As String = " "
    Public Property detalleResultado_UNOEE As String = " "
    Public Property urlWebServiceUNOEE As String = " "
    Public Property otrosDetalles As String = " "

    Public dtPaquetes As DataTable
    Public dtDocumentos As DataTable
    Public dtMovimientos As DataTable
    Public dtDescuentos As DataTable

End Class
